transformation/src/test/test-files/outputs/human_donor/part1.json

```diff
@@ -0,0 +1,2 @@
+{"id":"ENCDO845WKR","cross_references":["GEO:SAMN05897796"],"time_created":"2015-02-14T00:31:58.410757Z","age":"37","ethnicities":[],"organism":"/organisms/human/","sex":"male","award":"/awards/ENCODE/","lab":"/labs/encode-consortium/","life_stage":"adult","parent_ids":[],"status":"released","twin_ids":[],"submitted_by":"/users/6800d05f-7213-48b1-9ad8-254c73c5b83f/"}
```

Comment:
About `cross_references`:

It looks like this is actually following the same structure we use in ClinVar (assuming `GEO` is like a DB and `SAMN...` is like an ID). Would there be any value in aligning the struct formats?

Reply:
About `age`: Looks like it should be parse-able as a number after all!

Reply:
@raaidbroad or @quazi-broad do either of you have thoughts or opinions about how we store cross references? Would it make more sense to store them in a struct like we do for clinvar? In Encode, the schema is currently just storing them as strings.

Reply:
We can leave it as-is for now; the datatype of the column is a string either way, so if we decide to switch later we can do so without a schema migration.